CHANGELOG.md

```diff
@@ -1,4 +1,83 @@
-## 2.17.2 (Unreleased)
+## 2.18.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.3.2092**
+- Supported Terraform version: **v0.12.x** and **v0.13.x**
+
+### Features:
+1. Implemented support for BGP over GRE and BGP over LAN through ``enable_bgp_over_lan`` in **aviatrix_transit_gateway**, and following attributes in **aviatrix_transit_external_device_conn**:
+  - ``tunnel_protocol``
+  - ``remote_lan_ip``
+  - ``backup_remote_lan_ip``
+  - ``local_lan_ip``
+  - ``backup_local_lan_ip``
+  - ``remote_vpc_name``
+2. Implemented support for controller HTTPS certificate import with the following attributes in **aviatrix_controller_config**:
+  - ``ca_certificate_file_path``
+  - ``server_public_certificate_file_path``
+  - ``server_private_key_file_path``
+3. Implemented support for creating a Public Subnet Filtering gateway with the following attributes in **aviatrix_gateway**:
+  - ``enable_public_subnet_filtering``
+  - ``public_subnet_filtering_route_tables``
+  - ``public_subnet_filtering_ha_route_tables``
+  - ``public_subnet_filtering_guard_duty_enforced``
+4. Implemented support for configuring AWS Guard Duty:
+  - New resource **aviatrix_aws_guard_duty**
+  - New attribute ``aws_guard_duty_scanning_interval`` in **aviatrix_controller_config**
+5. Implemented support for configuring Learned CIDR Approval per connection:
+  - New attribute ``learned_cidrs_approval_mode`` in **aviatrix_transit_gateway**
+  - New attribute ``enable_learned_cidrs_approval`` in **aviatrix_device_transit_gateway_attachment**, **aviatrix_transit_external_device_conn** and **aviatrix_vgw_conn**
+6. Implemented support for configuring Manual Advertised CIDRs per connection:
+  - New attribute ``manual_bgp_advertised_cidrs`` in **aviatrix_device_transit_gateway_attachment**, **aviatrix_transit_external_device_conn** and **aviatrix_vgw_conn**
+7. Implemented support for FireNet with AWS Gateway Load Balancer (GWLB):
+  - New attribute ``enable_gateway_load_balancer`` in **aviatrix_transit_gateway**
+  - New attribute ``enable_native_gwlb`` in **aviatrix_vpc**
+  - Make ``firenet_gw_name`` Optional in **aviatrix_firewall_instance** and **aviatrix_firewall_instance_association**
+8. Implemented support for Monitor Gateway Subnets in **aviatrix_transit_gateway** and **aviatrix_spoke_gateway** with the following attributes:
```

Comment:
... support for Monitoring Gateway Subnets feature ... using the following attributes: